pyartcd/pyartcd/pipelines/comment_to_pr.py

```diff
@@ -277,27 +273,28 @@ def _builds_from_job(self):
         self._logger.debug(f"Response: {pprint.pformat(data)}")
 
         if data["count"] > 0:
-            distgits = []
             api_results = data["results"]
 
+            tasks = []
             for build in api_results:
                 if build["brew_task_state"] != "success":
                     # Do not check for unsuccessful builds
                     self._logger.info(f"Skipping Build {build['build_0_id']} because it failed.")
                     continue
 
-                distgit = self._get_distgit(build)
-                distgits.append(distgit)
+                tasks.append(asyncio.ensure_future(self._get_distgit(build)))
+
+            distgits = asyncio.gather(*tasks)
```

Comment:
Need to `await` this

Reply:
fixed in https://github.com/openshift/aos-cd-jobs/pull/3540/commits/5e04cb34c15013a011bc6fc5ef6507515c2e99ae